modules/oadp-backup-single-vm.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-backup-single-vm_{context}"]
+= Backing up a single VM
+
+If you have a namespace with many virtual machines (VMs), and want to back up only one of them, you can use the label selector to filter the VM that should be included in the backup. You can filter the VM by using the `app: vmname` label.
+
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have many VMs running in a namespace.
+* You have the added `kubevirt` plugin in the `DataProtectionApplication` (DPA) custom resource (CR).
+* The `DataProtectionApplication` CR is reconciled.
+* You have configured the `BackupStorageLocation` in the `DataProtectionApplication` CR and the `BackupStorageLocation` is available.
+
+.Procedure
+
+. Configure the `Backup` CR as shown in the following example:
++
+.Example `Backup` CR
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: vmbackupsingle
+  namespace: openshift-adp
+spec:
+  snapshotMoveData: true
+  includedNamespaces:
+  - <vm_namespace> # <1>
+  labelSelector:
+    matchLabels:
+      app: <vm_app_name> # <2>
```

Comment:
Don't you also need this here:
```
    - matchLabels:
        kubevirt.io/created-by: <datavolume_uid> # <1>
```

Reply:
from what i tested, no, that is why i think there is the reason i think there is  inconsistency in behavior
let me re-check and get some evidence

Reply:
Ahh ok. Yeah, I have a vague memory that for BIA additional items, only type filtering is done, not label filtering. On restore, it's a bit different, since I think the filtering is applied to the backup before starting restore, and restore additional items are limited to what's already included, since their main purpose is to declare an immediate dependency (this other thing needs to be restored before the current thing) rather than to bring new things into the restore.